hyperwallet/api.py

```diff
@@ -2684,3 +2685,110 @@ def listWebhookNotifications(self,
 
     def __buildUrl(self, *paths):
         return '/'.join(s.strip('/') for s in paths)
+
+    def uploadDocumentsForUser(self,
```

Comment:
It looks like you have added this method. Pls check if there is any code duplication and find out why its appearing so.

Reply:
v3 merged code